plugins/auth-node/package.json

```diff
@@ -25,8 +25,9 @@
   "dependencies": {
     "@backstage/backend-common": "^0.13.2-next.0",
     "@backstage/config": "^1.0.0",
+    "@backstage/catalog-model": "^0.13.0",
```

Comment:
Let's make this `1.0.1-next.0` as it's the latest version in the monorepo.

Reply:
Done